Model.cpp

```diff
@@ -769,7 +769,7 @@ int RunModel(Landscape* pLandscape, int seqsim)
 
 }
 
-#if RS_EMBARCADERO || LINUX_CLUSTER || RS_RCPP 
+#if RS_EMBARCADERO || LINUX_CLUSTER || RS_RCPP
```

Comment:
```suggestion
#if LINUX_CLUSTER || RS_RCPP
```

Reply:
We're not using Embarcadero anymore, might as well drop this!

Reply:
agreed